editor/src/components/canvas/dom-walker.ts

```diff
@@ -912,6 +912,7 @@ function getSpecialMeasurements(
     !positionValueIsDefault(elementStyle.right) ||
     !positionValueIsDefault(elementStyle.bottom) ||
     !positionValueIsDefault(elementStyle.left)
+  const hasTransform = elementStyle.transform !== 'none'
```

Comment:
Is `transform` never `undefined` or `null`?

Reply:
I checked the initial computed value, it is always 'none' https://developer.mozilla.org/en-US/docs/Web/CSS/transform